### DISCUSSION TITLE
Odroid XU4 - quale HA scaricare

### DISCUSSION THREAD

**Discussion by CaneCheRide** [indomus/forum#2462] [GENERALE - Discussioni generiche]
Ciao a tutti, volevo installare home assistant su di un odroid xu4 ma nelle informazioni di installazione trovo solo il sistema x le seguenti versioni: N2 - N2+ - C2 - C4 - M1
Mi sapete dire se una di queste è compatibile o dove posso trovare la versione per XU4? Con le versioni passate di home assistant era disponibile. Grazie mille

**Comment by roby70cr** [comment]
https://github.com/home-assistant/operating-system/releases

se scorri in basso trovi il file per XU4

https://github.com/home-assistant/operating-system/releases/download/10.3/haos_odroid-xu4-10.3.img.xz

(PS nel caso puoi modificare il link cambiando il tipo di modello con XU4

esempio
github.com/home-assistant/operating-system/releases/download/10.3/haos_odroid-**_m1_**-10.3.img.xz
github.com/home-assistant/operating-system/releases/download/10.3/haos_odroid-**_xu4_**-10.3.img.xz
